### ISSUE TITLE
Arglist tests failing.

### ISSUE THREAD

**Issue by richardlford** [dotnet/coreclr#3234]
The following tests are failing on Linux and OS X.

```
JIT/Directed/prefix/unaligned/1/arglist/arglist.sh
JIT/Directed/prefix/unaligned/2/arglist/arglist.sh
JIT/Directed/prefix/unaligned/4/arglist/arglist.sh
JIT/Directed/prefix/volatile/1/arglist/arglist.sh
```

I've added them to the non-Windows exclusion list.

These were not added recently, so something must have caused them to regress.

**Comment by RussKeldorph** [comment]
@sivarv Please investigate and triage/route appropriately.

**Comment by sivarv** [comment]
Failure of these test on unix/OSX is by design as vararg methods are not supported by CoreCLR on unix/OSX platform.

@richardlford - they were already disabled by an entry at the very top of  test\testsFailingOutsideWindows.txt. It looks like you have added them to that list again further down in your recent check-in.  Now they appear in that file listed twice!

Did you verify that they are running and passing in any of OSS CI runs?

**Comment by sivarv** [comment]
Marking this as test issue and assigning to @richardlford to update testsFailingOutsideWindows.txt so that these tests are not listed twice in that file.

**Comment by RussKeldorph** [comment]
When the lowercase ("prefix") exclusions are removed, these will start failing in the lab.  Presumably at least some of the Windows box(es) on which the tests are built have corrupted repos with messed up case.  I'm going to postpone this and revisit once builds happen end to end on Linux.

**Comment by RussKeldorph** [comment]
Fixed by #4296.
